### ISSUE TITLE
MacOS Crash when pressing CMD+]

### ISSUE THREAD

**Issue by Yohannfra** [ahrm/sioyek#471]
# Os version
Mac OS Monterey 12.4

# How to reproduce
Open any pdf and press **cmd**+**]**

# More informations

keys_user.config is empty

Thread 0 crash dump
```
Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   sioyek                        	       0x10d039bde DocumentView::goto_definition() + 30
1   sioyek                        	       0x10d05830c MainWidget::handle_command(Command const*, int) + 14540
2   sioyek                        	       0x10d05219e MainWidget::key_event(bool, QKeyEvent*) + 462
3   QtWidgets                     	       0x10ff5a9ef QWidget::event(QEvent*) + 527
4   QtWidgets                     	       0x10ff1ff5a QApplicationPrivate::notify_helper(QObject*, QEvent*) + 266
5   QtWidgets                     	       0x10ff2151b QApplication::notify(QObject*, QEvent*) + 907
6   QtCore                        	       0x1118a19f4 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 212
7   QtWidgets                     	       0x10ff79251 0x10ff0f000 + 434769
8   QtWidgets                     	       0x10ff78631 0x10ff0f000 + 431665
9   QtWidgets                     	       0x10ff1ff5a QApplicationPrivate::notify_helper(QObject*, QEvent*) + 266
10  QtWidgets                     	       0x10ff213e6 QApplication::notify(QObject*, QEvent*) + 598
11  QtCore                        	       0x1118a19f4 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 212
12  QtGui                         	       0x110ad8448 QGuiApplicationPrivate::processKeyEvent(QWindowSystemInterfacePrivate::KeyEvent*) + 152
13  QtGui                         	       0x110abd15b QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 219
14  QtGui                         	       0x110ab7f35 QWindowSystemInterface::flushWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 581
15  libqcocoa.dylib               	       0x1106a48e8 0x110673000 + 202984
16  libqcocoa.dylib               	       0x1106a4ab8 0x110673000 + 203448
17  AppKit                        	    0x7ff8081a71a6 -[NSWindow(NSEventRouting) _reallySendEvent:isDelayedEvent:] + 7150
18  AppKit                        	    0x7ff8081a539e -[NSWindow(NSEventRouting) sendEvent:] + 352
19  libqcocoa.dylib               	       0x1106a7ec5 0x110673000 + 216773
20  AppKit                        	    0x7ff8081a41c8 -[NSApplication(NSEvent) sendEvent:] + 2996
21  libqcocoa.dylib               	       0x1106aee6a 0x110673000 + 245354
22  AppKit                        	    0x7ff80845c59b -[NSApplication _handleEvent:] + 65
23  AppKit                        	    0x7ff808024f8e -[NSApplication run] + 623
24  libqcocoa.dylib               	       0x1106ab5d3 0x110673000 + 230867
25  QtCore                        	       0x11189da7f QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 431
26  QtCore                        	       0x1118a2002 QCoreApplication::exec() + 130
27  sioyek                        	       0x10d04af8c main + 4300
28  dyld                          	       0x11b60b51e start + 462

```

Thanks you

**Comment by ahrm** [comment]
I think this was a bug in version 1.5 . It has been fixed (but there are no official releases yet). Can you reproduce the issue on latest experimental builds here: https://github.com/hexomancer/sioyek/releases/tag/v0.31.395 ?

**Comment by Yohannfra** [comment]
Yes, exact same behavior.

Crash dump

```
Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   sioyek                        	       0x10b4475fe DocumentView::goto_definition() + 30
1   sioyek                        	       0x10b4670cb MainWidget::handle_command(Command const*, int) + 15339
2   sioyek                        	       0x10b4632cb MainWidget::handle_command_types(Command const*, int) + 267
3   sioyek                        	       0x10b460a6e MainWidget::key_event(bool, QKeyEvent*) + 478
4   QtWidgets                     	       0x10e37b9ef QWidget::event(QEvent*) + 527
5   QtWidgets                     	       0x10e340f5a QApplicationPrivate::notify_helper(QObject*, QEvent*) + 266
6   QtWidgets                     	       0x10e34251b QApplication::notify(QObject*, QEvent*) + 907
7   QtCore                        	       0x10fcc29f4 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 212
8   QtWidgets                     	       0x10e39a251 0x10e330000 + 434769
9   QtWidgets                     	       0x10e399631 0x10e330000 + 431665
10  QtWidgets                     	       0x10e340f5a QApplicationPrivate::notify_helper(QObject*, QEvent*) + 266
11  QtWidgets                     	       0x10e3423e6 QApplication::notify(QObject*, QEvent*) + 598
12  QtCore                        	       0x10fcc29f4 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 212
13  QtGui                         	       0x10eef9448 QGuiApplicationPrivate::processKeyEvent(QWindowSystemInterfacePrivate::KeyEvent*) + 152
14  QtGui                         	       0x10eede15b QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 219
15  QtGui                         	       0x10eed8f35 QWindowSystemInterface::flushWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 581
16  libqcocoa.dylib               	       0x10eac58e8 0x10ea94000 + 202984
17  libqcocoa.dylib               	       0x10eac5ab8 0x10ea94000 + 203448
18  AppKit                        	    0x7ff8081a71a6 -[NSWindow(NSEventRouting) _reallySendEvent:isDelayedEvent:] + 7150
19  AppKit                        	    0x7ff8081a539e -[NSWindow(NSEventRouting) sendEvent:] + 352
20  libqcocoa.dylib               	       0x10eac8ec5 0x10ea94000 + 216773
21  AppKit                        	    0x7ff8081a41c8 -[NSApplication(NSEvent) sendEvent:] + 2996
22  libqcocoa.dylib               	       0x10eacfe6a 0x10ea94000 + 245354
23  AppKit                        	    0x7ff80845c59b -[NSApplication _handleEvent:] + 65
24  AppKit                        	    0x7ff808024f8e -[NSApplication run] + 623
25  libqcocoa.dylib               	       0x10eacc5d3 0x10ea94000 + 230867
26  QtCore                        	       0x10fcbea7f QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 431
27  QtCore                        	       0x10fcc3002 QCoreApplication::exec() + 130
28  sioyek                        	       0x10b459417 main + 4311
29  dyld                          	       0x116b1651e start + 462

```

**Comment by ahrm** [comment]
Should be fixed in 9a6154aa0e73bd271c9912aa2a7d3825c5795586.

**Comment by hasantahir** [comment]
Can you please push the latest commit to the experimental build repository? Thanks.
